index.js

```diff
@@ -12,46 +12,45 @@ var options = {
 
 module.exports = function(privateKey, remoteIP, response, cb) {
   var error = null;
-  
+
   if (!privateKey) {
     error = 'Private key is required';
-  } else if (!remoteIP) {
```

Comment:
The only way I can think to do this is to wrap the key, ip and response into a params object and pass that in along with the callback (e.g. function(params, cb) {}). This seems slightly more complicated than just telling the user to pass in 'undefined' for the second parameter. I'm open to other suggestions though. The IP is optional: https://developers.google.com/recaptcha/docs/verify